buttercup/cogs/helpers.py

```diff
@@ -266,3 +268,10 @@ def parse_time_constraints(
     time_str = f"from {after_time_str} until {before_time_str}"
 
     return after_time, before_time, time_str
+
+
+def format_api_time(date_time: Optional[datetime]) -> Optional[str]:
+    """Format a date time to be used as an API parameter."""
+    # For some reason Blossom currently returns a normal HTML page

```

Comment:
Let's try and fix this first before building in workarounds

Reply:
This will be fixed with GreafeasGroup/blossom#203 and Buttercup uses the normal `isoformat` function now.